scripts/variantstore/wdl/GvsWithdrawSamples.wdl

```diff
@@ -0,0 +1,89 @@
+version 1.0
+
+workflow GvsWithdrawSamples {
+
+  input {
+    String dataset_name
+    String project_id
+
+    Array[String] sample_names
+
+    String? service_account_json_path
+  }
+
+  String sample_info_table = "sample_info"
```

Comment:
Is there a use case that benefits from `sample_info_table` being a declaration here (and an input in `WithdrawSamples`) or could its usage just be hardcoded `sample_info` in `WithdrawSamples`?